utenlandsopphold/domain/src/main/kotlin/no/nav/su/se/bakover/utenlandsopphold/domain/registrer/RegistrerUtenlandsoppholdHendelse.kt

```diff
@@ -102,7 +99,23 @@ data class RegistrerUtenlandsoppholdHendelse private constructor(
                 hendelsestidspunkt = hendelsestidspunkt,
                 versjon = forrigeVersjon,
                 meta = hendelseMetadata,
-            )
+            ).also {
+                require(it.entitetId == entitetId) {
+                    "Den persistert entitetId var ulik den utleda fra domenet:${it.entitetId} vs. $entitetId. "
+                }
+            }
         }
     }
 }
+
+fun RegistrertUtenlandsopphold.leggTil(hendelse: AnnullerUtenlandsoppholdHendelse): RegistrertUtenlandsopphold = RegistrertUtenlandsopphold.create(
```

Comment:
Kunne det passet bedre om vi la oss på en navnekonvensjon i baner av `hendelse()` eller `nyHendelse()`..  `leggTil()` gir litt andre assosiasjoner for meg.

Reply:
sure

Reply:
Jeg flyttet litt funksjonalitet fra funksjonen på sak til denne fila. Kalte funksjonene registrer/korriger/annuller